flowr/examples/word-count/DESCRIPTION.md

```
@@ -0,0 +1,35 @@
+
+Each line is read by `readline` and sent to `split` for tokenizing into words.
+Lines are processed in parallel — the splitter works on whichever line arrives
+next, demonstrating dataflow parallelism across input lines.
+
+A work counter tracks pending split operations, and a word counter accumulates
+the total. A `tap` gates the output until all splitting is complete (pending
+work reaches zero), then outputs the final word count to stdout.
+
+Root Diagram
+===
+<a href="root.dot.svg" target="_blank"><img src="root.dot.svg"></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)